tests/test_activities.py

```diff
@@ -1,10 +1,85 @@
+"""Test of Activities Module."""
 import json
```

Comment:
Did you introduce this changes? I thought this belonged to @jonathankamau .

Reply:
Which will be reflected once his work is merged first.